### DISCUSSION TITLE
Notion Pod

### DISCUSSION THREAD

**Discussion by Harshita-mindfire** [dendronhq/dendron#1364] [RFCs/Ideas]
This is the discussion for [RFC 30 - Notion Pod](https://wiki.dendron.so/notes/duBTaQ58NeFFsSDF4yoSs.html)

**Comment by bblearner** [comment]
Hello, is there any follow to this. I am interested in moving my data from notion to dendron.

I'm a software developer, happy to help with the development.

**Reply by nickolay-kondratyev** [reply]
Hi @bblearner Dendron is in maintaince mode. However, if you want to handle the integration of calling notion APIs I can give you a local srever API to create notes that are Dendron compatible.

**Reply by bblearner** [reply]
Hey @nickolay-kondratyev yeah I would be interested in creating a notion pod kind of thing. 
>if you want to handle the integration of calling notion APIs I can give you a local srever API to create notes that are Dendron compatible.

If I understand correctly, you want me to handle the "getting document from notion" part, and then we connect it to your API that can create Dendron compatible notes?
